index.html

```diff
@@ -4,42 +4,175 @@
+            <li class="stack">Ruby</li>
         </ul>
+        <a class="project">See Project</a>
+    </div>
+
+
+    <div class="second">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="grid-container2">
+            <li class="stack2">html</li>
+            <li class="stack2">bootstrap</li>
+            <li class="stack2">Ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="third">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fourth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fifth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
     </div>
 
+    <div class="sixth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
 </section>
 
+<section class="item-3">
+    <div class="about">
+
+        <h2 class="aboutMe">About me</h2>
+        <p class="decription">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="seeProject">Get My Resume</a>
```

Comment:
- [ ] Please apply the correct button background color as defined in the template

- [ ] Also, there should be a styling for the `:active` attribute in CSS as defined in the template:

![btns-styles](https://user-images.githubusercontent.com/36542986/134580398-37cf3669-842d-40e1-8620-abc1f6800534.jpg)

Reply:
Please this is for Desktop version